addons/l10n_ec/data/account_tax_template_withhold_profit_data.xml

```diff
@@ -0,0 +1,7510 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+		<!-- 
+		PURCHASE WITHHOLDS OVER ANTICIPADED PROFIT *IMPUESTO A LA RENTA
```

Comment:
The first thing an accountant does when evaluating Odoo is checking if all taxes are defined, so this can be a selling point. Most non essential taxes are deactivated, to be activated when needed. Maybe move them in a new module? l10n_ec_tax_extras ?

Reply:
This are everyday used in every purchase, local tax authority request that you issue a "withhold over vat and over tax profit" on every purchase for easier detecting of tax fraud.